AI/BattleAI/BattleAI.cpp

```diff
@@ -286,7 +301,7 @@ void CBattleAI::attemptCastingSpell()
 
 	for(PossibleSpellcast & psc : possibleCasts)
 		psc.value = evaluateSpellcast(psc);
-	auto pscValue = [] (const PossibleSpellcast &ps) -> int
+	auto pscValue = [](const PossibleSpellcast & ps) -> int

```

Comment:
I don't think Uncrustify provide any control over this, at least I didn't manage to find it.